lib/wine_o_inventory_api/businesses/store.ex

```diff
@@ -0,0 +1,23 @@
+defmodule WineOInventoryApi.Businesses.Store do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  schema "stores" do
```

Comment:
```suggestion
  schema "store" do
```

Reply:
@JeromeGoodrich Is your recommendation personal preference, conventional Ecto, both, something else? I suppose I'm fairly accustomed to the Rails convention of a plural table name but a singular ActiveRecord model name.

Reply:
I was under the impression table names are plural. I created the schema using the `mix phx.gen.schema` command in this doc; https://hexdocs.pm/phoenix/Mix.Tasks.Phx.Gen.Schema.html#content. According to the documentation, "The first argument is the schema module followed by its plural name (used as the table name)."

Reply:
At the end of the day, it is personal preference. 😅 I've just been burned in the past by inconsistencies between table names, some plural some singular and I generally think singular is easier to reason about as a convention. It's friendlier for international colleagues as well.

Reply:
@skp96 I wasn't aware of that convention in Phoenix. I would say go with the frameworks convention then. It's often a bad idea to try and work against it.